manifests/modelserving/tgis-ootb.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  labels:
+    opendatahub.io/dashboard: 'true'
+    opendatahub.io/ootb: 'true'
+  annotations:
+    description: Text Generation Inference Server (TGIS) is a high performance inference engine that deploys and serves Large Language Models.
+    openshift.io/display-name: TGIS Standalone ServingRuntime for KServe
+    openshift.io/provider-display-name: Red Hat, Inc.
+    tags: rhods,rhoai,kserve,servingruntime
+    template.openshift.io/documentation-url: https://github.com/opendatahub-io/text-generation-inference
+    template.openshift.io/long-description: This template defines resources needed to deploy TGIS standalone servingruntime with KServe in Red Hat OpenShift AI
+    opendatahub.io/modelServingSupport: '["single"]'
+  name: tgis-serving-template
+objects:
+- apiVersion: serving.kserve.io/v1alpha1
+  kind: ServingRuntime
+  metadata:
+    name: tgis-runtime
+    annotations:
+      openshift.io/display-name: TGIS Standalone ServingRuntime for KServe
+      opendatahub.io/recommended-accelerators: '["nvidia.com/gpu"]'
+  labels:
+    opendatahub.io/dashboard: 'true'
+  spec:
+    multiModel: false
+    supportedModelFormats:
+      - autoSelect: true
+        name: pytorch
+    containers:
+      - name: kserve-container
+        image: text-generation-inference
+        command: ["text-generation-launcher"]
+        args: 
+          - "--model-name=/mnt/models/"
+          - "--port=3000"
+          - "--grpc-port=8033"
+        env:
+          - name: TRANSFORMERS_CACHE
+            value: /tmp/transformers_cache
+        readinessProbe:
+          exec:
+            command:
+              - curl
+              - localhost:3000/health
+          initialDelaySeconds: 5
+        livenessProbe:
+          exec:
+            command:
+              - curl
```

Comment:
We probe the http server health probe because we currently cannot use grpc probes in knative, so these exec probes are a temporary workaround